packages/amplify-category-predictions/commands/predictions/update.js

```diff
@@ -0,0 +1,27 @@
+import { promptCategory } from '../../provider-utils/supportedPredictions';
+
+
+const subcommand = 'update';
+
+module.exports = {
+  name: subcommand,
+  alias: ['configure'],
+  run: async context => promptCategory()
+    .then((result) => {
```

Comment:
NON-BLOCKING: I feel if you are only using `predictionsCategory, provider, fileName` you can do a destruct from the result